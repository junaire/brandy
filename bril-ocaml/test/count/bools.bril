@main {
  a: int = const 0;
  b: bool = eq a a;
  c: bool = le a a;
}
