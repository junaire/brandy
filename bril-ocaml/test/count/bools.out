Ints: 1 Bools: 2
